src/components/CourseChecklist/CourseChecklist.scss

```diff
@@ -2,15 +2,42 @@
 @import "~bootstrap/scss/_buttons";
 @import "~bootstrap/scss/utilities/_screenreaders";
 
-$light-color: theme-color("light");
 $lightest-color: theme-color("lightest");
 $primary-color: theme-color("primary");
 $success-color: theme-color("success");
 
+.assignment-list-item {
+  list-style: none;
+  display: inline-block;
+
+  &:after { 
+    content: ","; 
+  }
+
+  &:last-child {
+    &:after {
+       content: ""; 
+    }
+  }
+}
+
+.assignment-list {
+  display: inline;
+  padding-inline-start: map-get($spacers, 3);
+}
+
 .checklist-item-link {
   text-decoration: none;
 }
 
+.checklist-item {
+  &:not(:hover) {
+    .checklist-item-link {
+      @include sr_only();
```

Comment:
This hides and shows the link on hover by using the `sr_only` Bootstrap utility.